spec/index.html

```diff
@@ -490,9 +490,9 @@ <h3>Literals</h3>
       <p>For details on numerical syntaxes (e.g., decimals, doubles), we refer to the <a data-cite="TURTLE#abbrev">RDF
           1.1: Turtle (Section 2.5.2)</a> specification.</p>
       <p>Other literals, such as strings, but also dates, binary, octal or hex code, XML or JSON code, or other types of
-        numbers (e.g., shorts), need to be written as <b>string literals</b>. If the literal is not a string, its
-        particular datatype must be indicated using the <code>^^</code> symbol and the corresponding <dfn
-          data-cite="XMLSCHEMA11-2#built-in-datatypes">datatype IRI</dfn>. For instance:</p>
+        numbers (e.g., shorts), need to be written as <b>datatyped literals</b>. These are represented as a string, 
+        followed by the `^^` symbol and the corresponding <dfn data-cite="XMLSCHEMA11-2#built-in-datatypes">datatype IRI</dfn>.
+        (`xsd:string` is the datatype IRI for strings; this is the default and can be omitted). For instance:</p>
```

Comment:
This will need some additional work. As it stands now, it suggests that `"test"^^` would be valid, as the string `"test"` is followed by `^^` and omits `xsd:string` (or more precisely, omits `http://www.w3.org/2001/XMLSchema#string`).